grafana/templates/grafana.ini.j2

```diff
@@ -286,7 +286,7 @@ editors_can_admin = true
 
 #################################### SMTP / Emailing ##########################
 [smtp]
-;enabled = false
+enabled = {{ grafana_smtp_enabled|lower }}
 ;host = localhost:25
```

Comment:
Can you please expose the host as a variable also? I image in most cases the desired smtp server will not be localhost. Probably will have to wrap it in an "if" so that it is only set if grafana_smtp_enabled is true

Reply:
I created an if condition block within the template to set the host. I set the default value to `localhost:25` for the ansible-roles repo.